src/protocol/ai.md

```diff
@@ -0,0 +1,175 @@
+---
+title: AI Framework Support
+description:
+  Comprehensive guide to AI framework compatibility and support on the iExec
+  platform
+---
+
+# 🤖 AI Framework Support
+
+The iExec Platform provides comprehensive support for popular AI and machine
+learning frameworks. Deploy your confidential AI applications with confidence
+using our tested and optimized framework configurations, all running in secure
```

Comment:
Alternative shorter proposition

Deploy confidential AI with ease. iExec supports popular AI/ML frameworks, running in secure Trusted Execution Environments (TEEs) with optimized configurations.